### ISSUE TITLE
Error on connect OpenDistro

### ISSUE THREAD

**Issue by fisico95** [ruflin/Elastica#2012]
When i wan't to connect with opendistro, on docker, i have this error message

Exception during establishing a SSL connection: javax.net.ssl.SSLHandshakeException: Insufficient buffer remaining for AEAD cipher fragment (2). Needs to be more than tag size (16)

What can i do?

**Comment by ruflin** [comment]
Elastica does not support OpenDistro only Elasticsearch. Maybe the above is related to https://github.com/ruflin/Elastica/issues/1731 ?

**Comment by coreation** [comment]
hey @ruflin this is not a feature request, but an honest open question, is there any move towards making support for OpenSearch, by you or any other contributors?

**Comment by ruflin** [comment]
@coreation Not really. The more the two projects diverge the harder it gets to support both. Supporting only Elasticsearch is already a lot of work and I prefer not to add more to my plate and the one of the community.

**Comment by coreation** [comment]
@ruflin  I completely understand and very much appreciate the work done so far. I was just nosing around for kNN-algorithms and saw OpenSearch was actively working on it but ElasticSearch itself wasn't (or at least not that I could find). Kudos to you, if I find an issue I can contribute on that we're missing in our usage I'll make sure to do so. I had the pleasure of already contributing a very small implementation a couple of years earlier ^^
